pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -619,38 +630,9 @@ func (v *vpnSeedServer) podTemplate(configMap *corev1.ConfigMap, secretCAVPN, se
 			},
 		}
 
-		if v.values.DisableNewVPN {
-			statusPath := filepath.Join(volumeMountPathStatusDir, "openvpn.status")
-			exporterContainer.Command = []string{
-				"/openvpn-exporter",
-				"-openvpn.status_paths",
-				statusPath,
-				"-web.listen-address",
-				fmt.Sprintf(":%d", metricsPort),
-			}
-			exporterContainer.Env = nil
-		}
-
 		template.Spec.Containers = append(template.Spec.Containers, exporterContainer)
 	}
 
-	if !v.values.DisableNewVPN {
-		template.Spec.InitContainers = []corev1.Container{
-			{
-				Name:            "setup",
-				Image:           v.values.ImageVPNSeedServer,
-				ImagePullPolicy: corev1.PullIfNotPresent,
-				Command: []string{
-					"/bin/vpn-server",
-					"setup",
-				},
-				SecurityContext: &corev1.SecurityContext{
-					Privileged: ptr.To(true),
-				},
-			},
-		}
-	}
```

Comment:
`!v.values.DisableNewVPN`: The `if` statement must be removed, but not the block itself.

Reply:
it was moved to the [template block above](https://github.com/gardener/gardener/pull/11714/files#diff-950bde462bb470c322897431b6bb1a22c933239d3ebf1c63fddb59f0cd3d6462R299-R312)

Reply:
Thanks for clarifying.